external_parser/unit_tests/test_example_joiner.cc

```diff
@@ -2,6 +2,58 @@
 #include "test_common.h"
 #include <boost/test/unit_test.hpp>
 
+BOOST_AUTO_TEST_CASE(example_joiner_test_ca) {
+  auto vw = VW::initialize("--quiet --binary_parser --cats 1 --min_value=0 --max_value=1", nullptr,
```

Comment:
if ca_v2.fb was generated from example_gen then I think this should be `--cats 4`

Reply:
Yes, also changed the other arguments to match example_gen